integration/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# we start from a Kafka image as the connector is in Kafka distribution
```

Comment:
This looks like the one true dockerfile we need - if it works as a demo, then we should bin the rest please